connector/signaltometricsconnector/testdata/configs/valid_full.yaml

```diff
@@ -0,0 +1,70 @@
+signaltometrics:
+  spans:
+    - name: span.exp_histogram
+      description: Exponential histogram
+      unit: us
+      include_resource_attributes:
+        - key: key.1
+          default_value: foo
+      attributes:
+        - key: key.2
+          default_value: bar
+      condition_sequence:
+        operator: or
+        conditions:
+          - attributes["some.optional.1"] != nil
+          - resource.attributes["some.optional.2"] != nil
```

Comment:
Is it also possible to have a single condition like `attributes["some.optional.1"] != nil OR resource.attributes["some.optional.2"] != nil`? That could be useful for more complex conditions where you need both `OR` and `AND` operators.

Reply:
Yes, you can do `attributes["some.optional.1"] != nil or resource.attributes["some.optional.2"] != nil` (`or` should be smaller case as per the grammar).